### PR TITLE
Implementation of Performance Native Ad Network

### PR DESCRIPTION
**Pull request by metin-cinar**
Performance Native is a native ad server that is used by some high volume sites that reach %75 of Turkey's internet traffic.

We want Performance Native Ad Server as a supported network for AMP ad component.

### CODE REVIEW

**Comment by metin-cinar** [comment]
Hi, when will the review process finish?

File: ads/performancenative.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```
Comment (torch2424) [inline ads/performancenative.js]: Incorrect copyright year 😄

File: ads/performancenative.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (torch2424) [inline ads/performancenative.md]: Incorrect Copyright year 😄

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,2277 @@
+{
```
Comment (torch2424) [inline package-lock.json]: Changes in this file should not be commited for this change 😄

File: package.json [hunk 4]
```diff
@@ -95,7 +95,7 @@
     "glob-move": "1.0.1",
     "globs-to-files": "1.0.0",
     "google-closure-compiler": "20190215.0.2",
-    "gulp": "3.9.1",
+    "gulp": "^3.9.1",
```
Comment (torch2424) [inline package.json]: Changes in this file should not be commited for this change 😄

File: yarn.lock [hunk 5]
```diff
@@ -6488,7 +6488,7 @@ gulp-webserver@0.9.1:
     tiny-lr "0.1.4"
     watch "^0.11.0"
 
-gulp@3.9.1:
+gulp@^3.9.1:
   version "3.9.1"
   resolved "https://registry.yarnpkg.com/gulp/-/gulp-3.9.1.tgz#571ce45928dd40af6514fc4011866016c13845b4"
```
Comment (torch2424) [inline yarn.lock]: Changes in this file should not be commited for this change 😄

**Pull request closed without merging**